src/main/scala/abalone/package.scala

```diff
@@ -3,8 +3,9 @@ import ornicar.scalalib
 
 package object abalone extends scalalib.Common with scalalib.OrnicarOption with scalalib.OrnicarBoolean {
 
-  val P1 = strategygames.Player.P1
-  val P2 = strategygames.Player.P2
+  // @TODO: check this was the correct way to handle the fact p1 is black and p2 is white
+  val P1 = strategygames.Player.P2
+  val P2 = strategygames.Player.P1
```

Comment:
this looks wrong to me, P1 should be Player.P1
In gameCollection is where i think the starting colour is defined etc.